apps/web/src/pages/settings/organization/OrganizationPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { PageContainer } from '@novu/design-system';
+import { useAuthContext } from '@novu/shared-web';
+import PageHeader from '../../../components/layout/components/PageHeader';
+import { css } from '../../../styled-system/css';
+import { HStack } from '../../../styled-system/jsx';
+import { OrganizationLogo } from './OrganizationLogo';
+import { OrganizationName } from './OrganizationName';
+
+const PAGE_TITLE = 'Organization profile';
+
+export function OrganizationPage() {
+  const { currentOrganization } = useAuthContext();
+
+  return (
+    <PageContainer title={PAGE_TITLE}>
+      <PageHeader title={PAGE_TITLE} />
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use `SettingsPageContainer`‏